src/keboola_mcp_server/clients/base.py

```diff
@@ -0,0 +1,338 @@
+from typing import Any, Optional, Union, cast
+
+import httpx
+
+JsonPrimitive = Union[int, float, str, bool, None]
+JsonDict = dict[str, Union[JsonPrimitive, 'JsonStruct']]
+JsonList = list[Union[JsonPrimitive, 'JsonStruct']]
+JsonStruct = Union[JsonDict, JsonList]
+
+
+class RawKeboolaClient:
+    """
+    Raw async client for Keboola services.
+
+    Implements the basic HTTP methods (GET, POST, PUT, DELETE)
+    and can be used to implement high-level functions in clients for individual services.
+    """
+
+    def __init__(
+        self,
+        base_api_url: str,
+        api_token: str,
+        headers: dict[str, Any] | None = None,
+        timeout: httpx.Timeout | None = None,
+    ) -> None:
+        self.base_api_url = base_api_url
+        self.headers = {
+            'Content-Type': 'application/json',
+            'Accept-encoding': 'gzip',
```

Comment:
```suggestion
            'Accept-Encoding': 'gzip',
```
case-insensitive, but let's use conventional spelling

Reply:
Fixed here: 7ddeea2928b6182864b3f3c5e9c4b7480be92aa9